packages/fl-exp/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "fl-exp",
+  "version": "1.0.3",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "bin": {
+    "fl": "./bin/cli.js"
+  },
+  "files": [
+    "bin",
+    "lib"
+  ],
+  "scripts": {
+    "compile": "tsc --build",
+    "build": "tsc --build",
+    "deploy": "cdk deploy",
+    "synth": "cdk synth",
+    "test": "jest",
+    "validate": "functionless",
+    "dev": "tsc --watch"
+  },
+  "dependencies": {
+    "@babel/core": "^7.19.3",
+    "@babel/plugin-transform-typescript": "^7.19.3",
+    "commander": "^9.4.0",
+    "dependency-tree": "^8.1.2",
+    "express": "^4.18.1",
+    "open": "^8.4.0",
+    "toposort": "^2.0.2"
+  },
+  "devDependencies": {
+    "@aws-sdk/client-dynamodb": "3.180.0",
+    "@aws-cdk/aws-appsync-alpha": "^2.43.1-alpha.0",
+    "@functionless/ast-reflection": "^0.3.1",
+    "@functionless/language-service": "^0.0.4",
+    "@swc/jest": "^0.2.22",
+    "@types/express": "^4.17.14",
+    "@types/jest": "^29.0.2",
+    "@types/node": "^18.7.18",
+    "@types/toposort": "^2.0.3",
+    "aws-cdk": "^2.43.1",
+    "aws-cdk-lib": "^2.43.1",
+    "aws-sdk": "^2.1215.0",
+    "constructs": "^10.1.111",
+    "esbuild": "^0.15.7",
+    "functionless": "^0.23.1",
```

Comment:
This dependency should be `"*"` to pick it up from the workspace.

Reply:
All of them or just `functionless`?

Reply:
Is it wise to have a public facing package use `*` in its dependencies? Or are we centrally managing them somewhere?